macros/udfs/business_hours.sql

```diff
@@ -0,0 +1,51 @@
+{% macro create_udf_business_hours() %}
+
+create or replace function {{target.schema}}.udf_business_hours(datetime1 TIMESTAMP_NTZ, datetime2 TIMESTAMP_NTZ)
+returns int
```

Comment:
Below, the final output values are rounded to 2 decimal points. Should the datatype returned be a decimal rather than an integer? If the decimal points end in `.00`, then `int` is ok, otherwise it'll be a decimal data type (at least in Snowflake).

Reply:
fixed